### PR TITLE
fixed sign error in mfd for CurrentLoop

### PR DESCRIPTION
**Pull request by bodokaiser**

### CODE REVIEW

File: test/magnetism.jl [hunk 1]
```diff
@@ -5,9 +5,40 @@
           CurrentLoop(1u"A", 10u"mm", -5u"mm")
 
     @testset "mfd" begin
-        current_loop = CurrentLoop(current = 1u"A", radius = 1u"m")
+        current_loop = CurrentLoop(current = 300u"A", radius = 39.6u"mm", height = 26.5u"mm")
```
Comment (roflmaostc) [inline test/magnetism.jl:8]: probably those tests test the faulty sign?

File: test/magnetism.jl [hunk 2]
```diff
@@ -5,9 +5,40 @@
...
+            (0u"mm", 5u"mm", 32.29u"Gauss"),
+            (0u"mm", 0u"mm", 27.31u"Gauss"),
+            (1u"mm", 0u"mm", 27.31u"Gauss"),
+            (2u"mm", 0u"mm", 27.31u"Gauss"),
+            (3u"mm", 0u"mm", 27.30u"Gauss"),
+            (4u"mm", 0u"mm", 27.30u"Gauss"),
+            (5u"mm", 0u"mm", 27.29u"Gauss"),
+            (6u"mm", 0u"mm", 27.28u"Gauss"),
+            (7u"mm", 0u"mm", 27.26u"Gauss"),
+            (8u"mm", 0u"mm", 27.26u"Gauss"),
```
Comment (roflmaostc) [inline test/magnetism.jl:31]: same result for different parameters?

File: test/magnetism.jl (hunk 1 above)
Comment (bodokaiser) [reply test/magnetism.jl:8]: Yes, they are to a specific problem of ours (this is why the values are so "special") for which I had the Comsol data

File: test/magnetism.jl (hunk 2 above)
Comment (bodokaiser) [reply test/magnetism.jl:31]: Yes, the magnetic field is very homogenous in the radial direction

**Pull request merged**